specification/cognitiveservices/data-plane/Language/preview/2022-03-01-preview/analyzeconversations.json

```diff
@@ -684,7 +684,7 @@
           ],
           "x-ms-enum": {
             "name": "ExtraInformationKind",
-            "modelAsString": false
+            "modelAsString": true
```

Comment:
Is this the only one you feel confident will never be in a response or will never change otherwise? Please bear in mind that my comments on one line typically apply to any similar instances throughout the file. Apologies if I didn't make that clear.

Reply:
This should be true actually.